.gitignore

```diff
@@ -3,3 +3,4 @@
 Cargo.lock
 
 .vscode/
+.idea/
```

Comment:
You probably add this to your personal ignore file: https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer.